### ISSUE TITLE
New Slack Channel - [#cobol-programming-course-pt]

### ISSUE THREAD

**Issue by n370** [openmainframeproject/foundation#210]
Please complete all the sections below to request a new channel. Channel requests can take a day or two, so please be patient :-).

# Channel name
`#cobol-programming-course-pt`

# Channel purpose
My dad and I recently discovered the Open Mainframe Project and got really excited about it. We signed up for the COBOL course and started going through the documentation available but we found it a bit hard to follow for beginners, specially non-native english speakers. We also found very little official and non-official learning material available elsewhere in portuguese which we both speak natively so we decided to publish a message for all portuguese speaking members in the **#general** channel (_picture bellow_) calling them to join us in learning and sharing experiences. We got a few reactions to that message and with that we decided to create a new channel **#study-group-pt** with the problem being we couldn't make it public so I'd like to have a public channel created so folks can join freely and participate. What really motivates us, particularly myself, is the possibilities that having a space like that open up for my dad to be able to share his extensive knowledge in mainframe systems for both myself and others to enjoy. He has over 30 years of experience in the I.T. industry with over 10 to 15 years of hands on experience coding software for Brazilian mainframe systems in his early days. He later moved on from engineering to management directing and managing US multinationals sales operations in the southeast of Brazil. As impressive as his curriculum may appear, unfortunately he still struggles with spoken and conversational english and that's when being able to communicate in portuguese becomes important for us. I hope this message finds its way and until then I'd like to wish all of you a nice day and thank you for your time.

<img width="961" alt="Screen Shot 2021-08-09 at 10 10 03" src="https://user-images.githubusercontent.com/1547648/128711413-4ead5028-10d1-4a35-b27e-31288b267ecb.png">

# Moderators for the channel
Dylson Valente Neto
- OMP Slack: @dylsonvalenteneto

Dylson Valente Filho
- OMP Slack: @dylsonvalentefilho

# Does this channel have a lifespan? ( if so, please let us know when that is so we can archive it or mark it inactive )

No, it should remain open for all the portuguese native speakers around the world to join it as the OMP slack grows. Administrators however can feel free to archive it if it ever becomes inactive.

**Comment by jmertic** [comment]
Added now - https://openmainframeproject.slack.com/archives/C02BA8YRY48
